src/main/java/com/redislabs/redisai/Model.java

```diff
@@ -0,0 +1,239 @@
+package com.redislabs.redisai;
+
+import com.redislabs.redisai.exceptions.JRedisAIRunTimeException;
+import java.util.ArrayList;
+import java.util.List;
+import redis.clients.jedis.Protocol;
+import redis.clients.jedis.util.SafeEncoder;
+
+/** Direct mapping to RedisAI Model */
+public class Model {
+  private Backend backend;
+  private Device device;
+  private String[] inputs;
+  private String[] outputs;
+  private byte[] blob;
+  private String tag;
+  private long batchSize;
+  private long minBatchSize;
+
+  /**
+   * @param backend - the backend for the model. can be one of TF, TFLITE, TORCH or ONNX
+   * @param device - the device that will execute the model. can be of CPU or GPU
+   * @param inputs - one or more names of the model's input nodes (applicable only for TensorFlow
+   *     models)
+   * @param outputs - one or more names of the model's output nodes (applicable only for TensorFlow
+   *     models)
+   * @param blob - the Protobuf-serialized model
+   */
+  public Model(Backend backend, Device device, String[] inputs, String[] outputs, byte[] blob) {
```

Comment:
this constructor should call the other constructor with  `tag = null, batchSize = 0, minBatchSize = 0`